glean-core/ffi/Cargo.toml

```diff
@@ -1,8 +1,12 @@
 [package]
 name = "glean-ffi"
-edition = "2018"
-version = "19.0.0"
+version = "20.0.0-alpha.0"
```

Comment:
Is there any way this could be inherited from the top level `Cargo.toml`?

Reply:
No, the two crates are independently versioned.
It's a bit of a bummer that there's no better support in cargo itself just yet, but I plan to paper over that with a bit more tooling.